src/components/Navbar/Navbar.jsx

```diff
@@ -54,7 +54,7 @@ const Navbar = () => {
               <div className="badge">
                 <i className="fas fa-shopping-cart bg-none grey"></i>
                 <div className="badge-number">
-                  {totalItems? <span>{totalItems}</span>:null}
+                  {cartState.cart.length? <span>{cartState.cart.length}</span>:null}
```

Comment:
you could also write this in this way
```jsx
{ cartState.cart.length && <span>{cartState.cart.length}</span> }
```

Reply:
Yes I was following what the blog said. What Atul posted islam the blog I followed